docs/concepts/pwa-building-blocks.md

```diff
@@ -7,6 +7,16 @@ kb_sync_latest_only
 
 # Building Blocks of the Intershop PWA Public Deployment
 
+- [Intershop Commerce Management (ICM)](#intershop-commerce-management-icm)
+- [PWA - Server-Side Rendering (SSR)](#pwa---server-side-rendering-ssr)
+- [PWA - Nginx](#pwa---nginx)
+- [Browser](#browser)
+- [Default Production Deployment](#default-production-deployment)
+- [Deployment Without Nginx](#deployment-without-nginx)
```

Comment:
Do the chapters in ll. 15-17 need to move one level down? Are these indeed subchapters of Default Production Deployment?

Reply:
I would say the levels are fine and leave it as it is for now.
Maybe the whole document might need some overhaul. But this is a different matter. ;)